### PR TITLE
Some more documentation.

### PR DESCRIPTION
**Pull request by maccoda**
Adding some documentation to assist with the set up with diesel, #6

### CODE REVIEW

File: diesel_setup.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Diesel setup
+These were the tasks I needed to perform before I could get diesel_cli installed.
```
Comment (steveklabnik) [inline diesel_setup.md]: could you put some newlines above and below this line please?

File: diesel_setup.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+## Ubunutu
+These are the packages referenced in the README that need to be installed for postgresql and sqlite 3:
```
Comment (steveklabnik) [inline diesel_setup.md]: same here, above and below. give it a little room to breathe 😄

**Comment by maccoda** [comment]
Will do 😁

**Comment by maccoda** [comment]
I also added a link to the instructions from the README, if not desired let me know and can easily change it back 👍

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ This web application shows people who have contributed to Rust.
 You'll need nightly Rust for now. This will change with Rust 1.15, when
 this will run on stable.
 
-Get the app set up. You'll need postgres installed. And sqlite3 headers I
+Get the app set up. You'll need [postgres](diesel_steup.md) installed. And [sqlite3 headers](diesel_steup.md) I
```
Comment (steveklabnik) [inline README.md]: you've mis-spelled setup here twice! 😄

File: diesel_setup.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+These are the packages referenced in the README that need to be installed for postgresql and sqlite 3:
+* `postgresql`
+* `postgresql-contrib`
+* `libpq-dev` (Not enough to just set up postgresql, need this for `libpq`)
+* `sqlite3`
+
+Then needed to set up postgresql using [this](https://help.ubuntu.com/community/PostgreSQL) tutorial.
+
+After this was able to install diesel_cli and then run the setup with the newly created credentials following the instructions in [README](README.md)
```
Comment (steveklabnik) [inline diesel_setup.md]: could you wrap this to 80 columns please?

File: diesel_setup.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Diesel setup
+
+These were the tasks I needed to perform before I could get diesel_cli installed.
+
+## Ubunutu
+
+These are the packages referenced in the README that need to be installed for postgresql and sqlite 3:
```
Comment (steveklabnik) [inline diesel_setup.md]: could you put a newline after this, please?

File: README.md (hunk 3 above)
Comment (maccoda) [inline README.md]: 😱 Oh no! Will fix that up!

File: diesel_setup.md (hunk 4 above)
Comment (maccoda) [inline diesel_setup.md]: Would you like all lines wrapped to 80 columns, inclusive of those with links?

**Pull request merged**